utils/Command/MissingInSetCommand.php

```diff
@@ -51,6 +52,7 @@ final class MissingInSetCommand extends Command
         IncreaseDeclareStrictTypesRector::class,
         StaticClosureRector::class,
         StaticArrowFunctionRector::class,
+        PostIncDecToPreIncDecRector::class,
```

Comment:
I think add to our own `rector.php` is better for this, so no need to exclude here, so we still have this functionality, but only on our `rector-src` ?

Reply:
I created PR:

- https://github.com/rectorphp/rector-src/pull/5923

for it.